src/legacy/core_plugins/kibana/public/management/sections/index_patterns/create_index_pattern_wizard/lib/get_indices.js

```diff
@@ -46,6 +46,7 @@ export async function getIndices(es, indexPatternCreationType, rawPattern, limit
   }
 
   const params = {
+    ignoreUnavailable: true,
```

Comment:
This addresses https://github.com/elastic/kibana/pull/27345#pullrequestreview-186843963

Requests without wildcards that failed to match were returning 500 errors. It seems that they should be returning 404 errors but its easier to apply this fix than track that down.

Reply:
same issue is described here - https://github.com/elastic/kibana/issues/27218